components/ringcentral/actions/send-sms/send-sms.mjs

```diff
@@ -1,64 +1,114 @@
+import ringcentral from "../../ringcentral.app.mjs";
+import utils from "../../common/utils.mjs";
 
 export default {
   key: "ringcentral-send-sms",
   name: "Send SMS",
-  description: "Creates and sends a new text message.",
-  version: "0.4.1",
+  description: "Creates and sends a new text message. See the API docs [here](https://developers.ringcentral.com/api-reference/SMS/createSMSMessage)",
+  version: "0.5.0",
   type: "action",
   props: {
-    ringcentral: {
-      type: "app",
-      app: "ringcentral",
+    ringcentral,
+    accountId: {
+      propDefinition: [
+        ringcentral,
+        "accountId",
+      ],
     },
-    serverURL: {
+    extensionId: {
+      propDefinition: [
+        ringcentral,
+        "extensionId",
+      ],
+      description: "Internal identifier of an extension.",
+    },
+    fromPhoneNumber: {
       type: "string",
-      description: "The base endpoint host used for the RingCentral API.",
+      label: "Phone Number",
+      description: "The phoneNumber for the sender of an SMS message. Property must be filled to correspond to one of the account phone numbers which is allowed to send SMS. Phone number in E.164 format",
+    },
+    toPhoneNumbers: {
+      type: "string[]",
+      label: "To Phone Numbers",
+      description: "Phone number of receiver(s) of an SMS message. Phone number in E.164 format.",
     },
-    account_id: {
+    text: {
       type: "string",
-      description: "Internal identifier of a RingCentral account.",
+      description: "Text of a message. Max length is 1000 symbols (2-byte UTF-16 encoded). If a character is encoded in 4 bytes in UTF-16 it is treated as 2 characters, thus restricting the maximum message length to 500 symbols.",
     },
-    extension_id: {
+    countryId: {
       type: "string",
-      description: "Internal identifier of an extension.",
+      label: "Country ID",
+      description: "Internal identifier of a country.",
+      optional: true,
     },
-    phone_number: {
+    countryUri: {
       type: "string",
-      description: "The phoneNumber for the sender of an SMS message. Property must be filled to correspond to one of the account phone numbers which is allowed to send SMS.",
+      label: "Country URI",
+      description: "Canonical URI of a country.",
+      optional: true,
     },
-    to: {
-      type: "any",
-      description: "Receiver of an SMS message. The phoneNumber property must be filled.",
+    countryName: {
+      type: "string",
+      label: "Country Name",
+      description: "Official name of a country.",
+      optional: true,
     },
-    text: {
+    countryIsoCode: {
       type: "string",
-      description: "Text of a message. Max length is 1000 symbols (2-byte UTF-16 encoded). If a character is encoded in 4 bytes in UTF-16 it is treated as 2 characters, thus restricting the maximum message length to 500 symbols.",
+      label: "Country ISO Code",
+      description: "ISO code of a country.",
```

Comment:
Can you add a ref to ISO country code, please? Maybe this can be an `async option`?

Reply:
Actually I had to remove these `country` fields except `countryId` because I noticed I  was getting request errors like
```
Parameter [uri] value is invalid.
Parameter [name] value is invalid.
Parameter [isoCode] value is invalid.
Parameter [callingCode] value is invalid.
```